### DISCUSSION TITLE
Urgent please help me

### DISCUSSION THREAD

**Discussion by NeonXfury** [2bllw8/anemo#63] [Q&A]
Please help me, I have forgotten my anemo storage password, I recently lost my close one and I had saved the photo of the person there, I only have the one copy, please help me.... How can I access those files aniket.personalmail19@gmail.com is my contact email please help me .... I will be thankful to you!! I just want to recover that only file for once!!

**Comment by ghost** [comment]
sorry for your loss, but it's impossible to recover that file without the password.

**Reply by NeonXfury** [reply]
Brother isn't there any workaround? I am okay with paying too for the help!! Please I just need it so desperately

**Comment by pixel2user** [comment]
NeonXfury - Joined 34 minutes ago

**Reply by nain-F49FF806** [reply]
The account does look fishy/unusual for sure. Thanks for noting. 
Although it could simply mean they are not a dev (or related) and just desperately made an account here.

**Comment by pixel2user** [comment]
If it is possible to easily get around the password I wouldn't use the app any longer.

**Comment by NeonXfury** [comment]
Brother please!! Isn't there a single way??

**Comment by nain-F49FF806** [comment]
@NeonXfury Don't panic. First of all, the important thing is to not uninstall the app. 
If you do that, all the data will be lost with no possibility of recovery. 

Next, answers to these questions might help:
- Is the device rooted?
- Manufacturer of the device

If you own the device, and it is rooted then it should be possible to recover quite easily. 

If not rooted, then depending on manufacturer and os version, 
there may be un-patched exploits that could be.. well "exploited". To help you get your data back.
